src/components/Basic/Modal/Modal.module.scss

```diff
@@ -0,0 +1,60 @@
+.modal {

```

Comment:
Does it make sense to have a high z-index on the modal, so that other absolutely positioned elements don't go above it?

Reply:
Added z-index 1000